app/controllers/admin/employees_controller.rb

```diff
@@ -4,12 +4,13 @@ module Admin
   class EmployeesController < AdminController
     # GET /admin/employees
     def index
-      @employees = Employee.all.includes([:orders])
+      @employees = Employee.all.includes([:orders]).sort
     end
 
     # GET /admin/employees/1
     def show
       find_employee
+      @orders = Order.all.where(employee: @employee)
```

Comment:
why not - you can also get rid of that, and simply call `@employee.orders` in the view
```suggestion
      @orders = @employee.orders
```

Reply:
Thanks! Changed ;)